cmd/ddev/cmd/clean.go

```diff
@@ -62,28 +64,21 @@ Additional commands that can help clean up resources:
 			os.Exit(1)
 		}
 
-		if util.Confirm("Confirm removal of the listed items?") {
-			ddevDir := globalconfig.GetGlobalDdevDir()
-			_ = os.RemoveAll(filepath.Join(ddevDir, "testcache"))
-			_ = os.RemoveAll(filepath.Join(ddevDir, "bin"))
-			_ = os.RemoveAll(filepath.Join(ddevDir, "downloads"))
+		confirm := util.Prompt("Are you sure you want to continue? Y/N", "N")
+		if confirm == "Y" {
+			globalDdevDir := globalconfig.GetGlobalDdevDir()
+			_ = os.RemoveAll(filepath.Join(globalDdevDir, "testcache"))
+			_ = os.RemoveAll(filepath.Join(globalDdevDir, "bin"))
 
 			output.UserOut.Print("Deleting snapshots and downloads for selected projects...")
 			for _, project := range projects {
 				// Delete snapshots and downloads for each project
-				_ = os.RemoveAll(filepath.Join(project.AppRoot, ".ddev", "downloads"))
```

Comment:
Let's check the error and `util.Warn()` about it, but not fail. If we found it and can't delete it, there's something going on.

Reply:
I will add the error check.